<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bfvnoticebotClass</class>
 <widget class="QMainWindow" name="bfvnoticebotClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>战地五公告机</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/bfvnoticebot/bfv.ico"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QRadioButton" name="apiRadioButton">
               <property name="text">
                <string>社区API</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="napcatRadioButton">
               <property name="text">
                <string>NapCat</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>每轮喊话的间隔 秒</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="roundSpinBox">
                 <property name="minimum">
                  <number>20</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>120</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>单次发多条文本的间隔 秒</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="multiSpinBox">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                 <property name="value">
                  <number>2</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="actionNapcatConfig"/>
    <addaction name="actionApiConfig"/>
   </widget>
   <widget class="QMenu" name="menuothers">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>其它</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuConfigure"/>
   <addaction name="menuothers"/>
  </widget>
  <action name="actionNapcatMode">
   <property name="text">
    <string>napcat</string>
   </property>
  </action>
  <action name="actionApiMode">
   <property name="text">
    <string>社区api</string>
   </property>
  </action>
  <action name="actionNapcatConfig">
   <property name="text">
    <string>NapCat</string>
   </property>
  </action>
  <action name="actionApiConfig">
   <property name="text">
    <string>社区API</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bfvnoticebot.qrc"/>
 </resources>
 <connections/>
</ui>
